src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js

```diff
@@ -165,4 +178,33 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
         this.speed = null;
         this.capability = null;
     }
+
+    /**
+     * Build the string used for `#icaoWithWeightClass`
+     *
+     * @for AircraftTypeDefinitionModel
+     * @method _buildIcaoWithWeightClass
+     * @return {string}
+     * @private
+     */
+    _buildIcaoWithWeightClass() {
+        const HEAVY_LETTER = 'H';
+        const SUPER_LETTER = 'U';
+        let aircraftIcao = this.icao;
+
+        switch (this.weightclass) {
+            case HEAVY_LETTER:
+                aircraftIcao = `${HEAVY_LETTER}/${this.icao}`;
+
+                break;
+            case SUPER_LETTER:
+                aircraftIcao = `${SUPER_LETTER}/${this.icao}`;
```

Comment:
In our simulators, the strips actually also used the `H` prefix even for the super weightclass. That said, I don't know if that's correct; I can't find any pictures of A380 flight strips and the only person I know who could answer this was not sure, though he indicated he thought with a low level of confidence that it might actually be `J`.

For now, let's stick with the heavy letter when the weight class is super.